trulens_eval/examples/dummy_example.ipynb

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            print(f"thread count={threading.active_count()}, promises={TP().promises.qsize()}", end="")
</pre><p>Can you add a quick markdown cell explaining what you're doing here? Specifically the promises?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/418/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>